src/Games/NexusMods.Games.MountAndBlade2Bannerlord/Hack.cs

```diff
@@ -0,0 +1,747 @@
+using System.Collections;
+using System.Xml;
+using Bannerlord.ModuleManager;
+namespace NexusMods.Games.MountAndBlade2Bannerlord;
+
+/// <summary>
+/// Temporary code, to make game 'usable' while we wait for approval to make the game
+/// properly usable. Scheduled for DELETION.
+/// </summary>
+public static class Hack
```

Comment:
I hate this as much as you probably do, we should really implement something to recognize and handle game/external files.

Reply:
Yup, it's the best I can do, given the limitations I've been presented with of not ingesting.